src/gateway/environments/jsdom-sixteen/__tests__/jsdom-sixteen-environment.test.js

```diff
@@ -1100,6 +1101,7 @@ window["__register__"](fakeRender);
             // Assert
             expect(fakeWindow.close).toHaveBeenCalled();
             expect(fakeGate.close).toHaveBeenCalled();
+            expect(fakeResourceLoader.close).toHaveBeenCalled();
```

Comment:
This ensures we check this is working, so we won't miss this if it happens again.